lib/sql_enum/enum_type.rb

```diff
@@ -0,0 +1,12 @@
+module SqlEnum
+  # EnumType that always returns a symbol
+  class EnumType < ActiveRecord::Enum::EnumType
+    def cast(...)
```

Comment:
Is everything on 2.7 now?

Reply:
Ya that is a good question... Probably safer to just use normal args

Reply:
Switched to a normal arg